projects/packages/my-jetpack/src/class-initializer.php

```diff
@@ -168,6 +168,7 @@ public static function enqueue_scripts() {
 				'fileSystemWriteAccess' => self::has_file_system_write_access(),
 				'loadAddLicenseScreen'  => self::is_licensing_ui_enabled(),
 				'adminUrl'              => esc_url( admin_url() ),
+				'isJetpackPluginActive' => Plugins_Installer::is_plugin_active( 'jetpack/jetpack.php' ) || Plugins_Installer::is_plugin_active( 'jetpack-dev/jetpack.php' ),
```

Comment:
@retrofox  I noticed that we require the same behavior in backup, scan, and also in VideoPress according to this task: https://github.com/Automattic/jetpack/issues/29593, and all of them are returning to My Jetpack instead of going to the recommendations page.

Given that My Jetpack is the shared package among them, and is the one package doing it differently, we suggest including it here.

Unfortunately, I wasn't able to reproduce the VideoPress scenario because I didn't see the option to actually purchase from Jetpack, just to add/activate the plugin.

What do you think?

Reply:
I posted mine after yours @Initsogar ... But if we want to move everyone to `recommendations`, I think this is probably the way. Should we just confirm this with design?

Reply:
Thanks for the feedback @renatoagds and @retrofox. Valuable insights!

I just requested feedback from design to understand the upgrade flow approach through My Jetpack. Noticed that for Jetpack Search they are asking for something a bit different in https://github.com/Automattic/jetpack/issues/29592 so it would be great to clarify.

It would be great to clarify that before making any changes. Either way, your approach looks neat @renatoagds!

p6TEKc-6ZE-p2